code/src/components/NewThoughts.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const NewThoughts =({ newMessage, onNewMessageChange, onFormSubmit }) => {
+  return (
+    <label>
+      <form
+        className='message-section'
+        onSubmit={onFormSubmit}>
+        <label
+          htmlFor='newMessage'>
+          <h2>What's making you happy right now?</h2>
+        </label>
+        <input
+          className='message-area'
+          id='messageNew'
+          type='text'
+          value={newMessage}
+          onChange={onNewMessageChange}
+        />
```

Comment:
I think htmlFor and id got mixed up a bit so they don't match anymore 😵. Set both of them to either "newMessage" or "messageNew" to improve accessability!